app/controllers/v0/sessions_controller.rb

```diff
@@ -58,55 +45,33 @@ def new
               SAML::SettingsService.idme_loa3_url(current_user, success_relay: params[:success_relay])
             when 'slo'
               authenticate
-              destroy_sso_cookie!
-              SAML::SettingsService.slo_url(session)
+              SAML::SettingsService.logout_url(session)
             end
       render json: { url: url }
     end
     # rubocop:enable Metrics/CyclomaticComplexity
 
-    # Member Action: auth token required
-    # method is to opt in to MFA for those users who opted out
-    # authn_context is the policy, connect represents the ID.me flow
-    # TODO: DEPRECATED
-    def multifactor
-      render json: {
-        multifactor_url: SAML::SettingsService.mfa_url(current_user, success_relay: params[:success_relay])
-      }
-    end
-
-    # Member Action: auth token required
-    # method is to verify LOA3. It is not necessary to verify for DSLogon or MHV who are PREMIUM users.
-    # These sign-in users return LOA3 from the auth_url flow.
-    # TODO: DEPRECATED
-    def identity_proof
-      render json: {
-        identity_proof_url: SAML::SettingsService.idme_loa3_url(current_user, success_relay: params[:success_relay])
-      }
-    end
-
-    # TODO: DEPRECATED
-    def destroy
-      render json: { logout_via_get: SAML::SettingsService.slo_url(session) }, status: 202
+    def logout
```

Comment:
does this require front end changes?

Reply:
Good question, but, no, it shouldn't. That's the beauty of how the FE is fetching the links from new/slo... it fetches the link, but instead of that link being id.me it's not going to be vets.gov, and vets.gov just turns around and does a redirect after destroying the session.

Reply:
we definitely need to test it on staging though -- i'm not satisfied with it having worked on local alone.

Reply:
So to be explicit, the steps are:

1. Website, upon clicking 'Sign Out', gets the logout URL with `/sessions/new/slo` and opens the new window for it.
2. Logout action destroys the session, and the chain of redirects will all occur in that new window, starting with the redirect to ID.me.
3. As it did before, the flow will eventually end up at the website's logout screen.

Reply:
yup, i tested it locally, it all works, you're welcome to try too, but the real test will be on staging as we've ran into problems attempting this before. I think passing the base64encoded session token solves the authentication problem we had before though... 

@jkassemi do you have problem with that? I know previously you were not ok with session token getting cached in browser history. What about for this one instance, and only as base64_urlsafe encoded? I know technically it's still there, but I'm thinking this little obfuscation is helpful.

Reply:
Thanks for pulling me in here! Yes, and you may have answered this in slack already - why do we need to return the URL that the client must then follow to log out? Can we just link to the destroy action directly and let it handle the redirect at the same time without juggling the session token?